odl/solvers/advanced/__init__.py

```diff
@@ -19,10 +19,14 @@
 
 __all__ = ()
 
+from . import proximal_operators
+from .proximal_operators import *
+__all__ += proximal_operators.__all__
+
 from . import chambolle_pock
 from .chambolle_pock import *
 __all__ += chambolle_pock.__all__
 
-from . import proximal_operators
-from .proximal_operators import *
-__all__ += proximal_operators.__all__
+from . import douglas_rachford_primal_dual
```

Comment:
`douglas_rachford` is enough for my taste. Chambolle-Pock is also a primal-dual method.